### PR TITLE
Iss13 navbar dashboard

### PR DESCRIPTION
**Pull request by abohakobyan**
UserContext,  Navbar, placeholder FAQ, Dashboard, Blog

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -1,20 +1,76 @@
...
   return (
+    <UserContext.Provider value={value}>
     <MuiThemeProvider theme={theme}>
       <BrowserRouter>
-        <Route path="/" component={LandingPage} />
+        <Route exact path="/" render={() => <Redirect to="/signup" />} />
+        <Route path="/signup" component={SignUp} />
+        <Route path="/login" component={Login} />
+        <Route path="/background" component={Background} />
+        <Navbar/>
```
Comment (skassam21) [inline client/src/App.js]: Will the navbar show up at the bottom of the routes above ^ You may need to put the navbar into another route and prefix all the dashboard pages with /dashboard

File: server/users/user.service.js [hunk 2]
```diff
@@ -30,7 +30,11 @@ async function getAll() {
 }
 
 async function getById(id) {
-  return await User.findById(id);
+  const user = await User.findById(id)
+  if(user)
+  return {...user.toJSON()};
```
Comment (skassam21) [inline server/users/user.service.js:35]: instead, you can also make a toObject method and it'll convert it automatically as well

File: client/src/pages/Blog.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import React, { useContext} from "react";
+import {UserContext}from '../contexts/UserContext'
+
+
+function Blog(props){
+  
+  const {user} = useContext(UserContext)
```
Comment (skassam21) [inline client/src/pages/Blog.js]: Do you have a linter set up? Notice there is inconsistencies in terms of semicolons and spacing

**Review by skassam21 — COMMENTED** [review]
Looks good and generally works - a few comments

**Review by samsoncsyu7777 — APPROVED** [review]

**Pull request merged**